### PR TITLE
golint fixes

### PR DESCRIPTION
**Pull request by afinkenstadt**
restructured code per #290

### CODE REVIEW

File: httputil/respbuf.go [hunk 1]
```diff
@@ -12,20 +12,26 @@ import (
...
+// ResponseBuffer is the current response being composed by its owner.
+// It implements the http.ResponseWriter interfaces: Write, WriteHeader, and Header,
+// as well as io.WriterTo interface.
 type ResponseBuffer struct {
 	buf    bytes.Buffer
 	status int
 	header http.Header
 }
 
+// Write implements the ResponseWriter interface on a ResponseBuffer
```
Comment (adg) [inline httputil/respbuf.go]: please remove "on a ResponseBuffer" and add a period to the end of these comments.

File: httputil/respbuf.go [hunk 2]
```diff
@@ -12,20 +12,26 @@ import (
 	"strconv"
 )
 
+// ResponseBuffer is the current response being composed by its owner.
+// It implements the http.ResponseWriter interfaces: Write, WriteHeader, and Header,
```
Comment (adg) [inline httputil/respbuf.go]: It implements http.ResponseWriter and io.WriterTo.

**Comment by adg** [comment]
We have switched to Gerrit for Code Review. I have created a Gerrit code review that contains the (attributed) contents of this change: https://go-review.googlesource.com/c/20464

**Comment by shurcooL** [comment]
@adg, that link is 404, is something missing?

**Pull request closed without merging**